doc/shared/Fortran/Introduction.rst

```diff
@@ -180,19 +180,19 @@ team.
 Installation
 ------------
 
-The installation procedure for the Fortran interfaces is the same as for the
-C/C++ core of SUNDIALS, refer to :numref:`Installation`.  The CMake option to
-turn on the Fortran interfaces in a SUNDIALS build is
-:cmakeop:`BUILD_FORTRAN_MODULE_INTERFACE`.  The Spack variant is ``+fortran``.
+The installation procedure for the Fortran interfaces is the same as for the C/C++ core of SUNDIALS, refer
+to :numref:`Installation`.
+The CMake option to turn on the Fortran interfaces in a SUNDIALS build is :cmakeop:`BUILD_FORTRAN_MODULE_INTERFACE`.\
+The Spack variant is ``+fortran``.
 
 .. _Fortran.Portability:
 
 Important notes on portability
 ------------------------------
-
-The SUNDIALS Fortran 2003 interface *should* be compatible with any compiler
-supporting the Fortran 2003 ISO standard.
-
+The installation procedure for the Fortran interfaces is the same as for the
+C/C++ core of SUNDIALS, refer to :numref:`Installation`.  The CMake option to
+turn on the Fortran interfaces in a SUNDIALS build is
+:cmakeop:`BUILD_FORTRAN_MODULE_INTERFACE`.  The Spack variant is ``+fortran``.
```

Comment:
Is this text supposed to replace what was here before and duplicate lines 183-186?

Reply:
This duplicate text should be removed.  I think the apply suggestion somehow put it in the wrong spot.